### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_ScreenReader: JAWS does not read the "No results found" content as soon as searching is performed and no results are found.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52269]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while JAWS is ON.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to "Search" textbox. Enter any value say "abc" and hit "Enter".

**Actual:**
JAWS does not read the "No results found....." content which tells user the search result did not produce any valid result.

**Expected:**
JAWS should read the "No results found...." content as soon as search is performed by user and no outputs are found.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader users will not know about the empty search content and other meaningful information.When "No results...." content is read ,the users will then and there come to know about the search results output.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ViewSearch_JAWS_NoResultsFound.pptx](https://github.com/Microsoft/vscode/files/2114195/A11y_VSCode_ViewSearch_JAWS_NoResultsFound.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
This works for VoiceOver and NVDA at least.

**Comment by isidorn** [comment]
We do not support jaws
